CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Print all values, including object specifiers, readably.
```

Comment:
Can you insert the issue numbers here, or do you want to after merge?

Reply:
Now works!